apps/public-docsite/src/components/Site/Site.tsx

```diff
@@ -96,7 +96,7 @@ export class Site<TPlatforms extends string = string> extends React.Component<
     let platform = 'default' as TPlatforms;
 
     // If current page doesn't have pages for the active platform, switch to its first platform.
-    if (Object.keys(navData.pagePlatforms).length > 0 && navData.activePages.length === 0) {
+    if (Object.keys(navData.pagePlatforms!).length > 0 && navData.activePages!.length === 0) {
```

Comment:
all these nullable issues were exposed by no strict null checks, while using `!` is deprecated and instead `?` should be used, to be able to not break anything website related, this is the safest approach.